From 8739cd6f5fe0f2ed0cf9913170573c62ee246c63 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Burkhard=20L=C3=BCck?= <lueck@hube-lueck.de>
Date: Thu, 22 Sep 2016 21:17:07 +0200
Subject: cleanup CMakeLists.txt

remove qt5_wrap_ui wrong because the app uses xgettext and line 60 already has ki18n_wrap_ui
remove add_definitions (-DTRANSLATION_DOMAIN=SymbolEditor), that is only for libraries
and
src/Main.cpp:84:    KLocalizedString::setApplicationDomain(SymbolEditor);
is used to load the translation catalog
REVIEW:128126
---
 CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d1016cb..d88174a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -64,10 +64,6 @@ add_subdirectory (doc)
 
 kconfig_add_kcfg_files (SymbolEditor_SRCS SymbolEditor.kcfgc)
 
-qt5_wrap_ui (SymbolEditor_SRCS
-    ui/EditorConfigPage.ui
-)
-
 ecm_install_icons (ICONS sc-apps-SymbolEditor.svgz DESTINATION ${ICON_INSTALL_DIR})
 
 add_executable (SymbolEditor ${SymbolEditor_SRCS})
@@ -89,8 +85,6 @@ add_definitions (
     -DKDE_NO_COMPAT
 )
 
-add_definitions (-DTRANSLATION_DOMAIN="SymbolEditor")
-
 if (SILENCE_DEPRECATED)
     add_definitions( -Wno-deprecated-declarations )
 endif (SILENCE_DEPRECATED)
-- 
cgit v1.1
